spec/queries/find_uncaptioned_members_spec.rb

```diff
@@ -4,13 +4,32 @@
 RSpec.describe FindUncaptionedMembers do
```

Comment:
> Can we add a test demonstrating that it doesn't find anything if `captions_required` is false? Because that logic in the file_set model so it's not obvious from reading the query itself.

Yup, done!